src/components/Layout/Header/GlobalSearch/GlobalSearchButton.tsx

```diff
@@ -79,9 +86,38 @@ export const GlobalSeachButton = () => {
     }
   })
 
+  const debouncedParseAddressInput = useMemo(() => {
+    return debounce(
+      () => {
+        if (!searchQuery?.length) return
+        setSendResults([])
+        ;(async () => {
+          const parsed = await parseAddressInput({ value: searchQuery })
+          if (parsed) {
+            // TODO: handle multiple possible ChainIds for a given address
+            setSendResults([
+              {
+                type: GlobalSearchResultType.Send,
+                id: getChainAdapterManager().get(parsed.chainId)!.getFeeAssetId(),
+              },
+            ])
+          }
+        })()
```

Comment:
Only send results for now, but more contextual actions to come e.g "View in Explorer"